packages/render/src/ui/frame.base.tsx

```diff
@@ -259,40 +287,19 @@ export function BaseFrameUI<TStylingProps extends Record<string, unknown>>({
       buttonsContainer:
         frameUiState.status === "loading" ||
         !frameUiState.frame.buttons ||
-        frameUiState.frame.buttons.length === 0
+        frameUiState.frame.buttons.length === 0 ||
+        !buttonsProps
```

Comment:
the conditions are pretty much the same here, could just do with a `buttonProps && components....`?